src/components/attendee/ProfileLinkIcon/ProfileLinkIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from "react";
+import { UseFormSetValue } from "react-hook-form";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { UserProfileModalFormData } from "types/profileModal";
+
+import { useProfileModalLinkIcon } from "utils/profileModalLinkUtilities";
+
+import CN from "./ProfileLinkIcon.module.scss";
```

Comment:
I think our newest code style is to name them `styles` @goran-peoski-work Can you verify this?

Reply:
[04f5003](https://github.com/sparkletown/sparkle/pull/3027/commits/04f5003da32c5d46ad3c388886090803250bdac9)